docs/usage/usage-applicants.md

```diff
@@ -0,0 +1,96 @@
+# Usage for applicants
+
+This section describes the typical usage of the funding portal for persons who apply to the funding program. This includes the application process as well as the workflow after a funding has been granted.
+
+The actions described below are grouped into the various [phases](./usage.md#phases-of-a-funding-case) that a funding case can go through.
+
+## Funding Portal
+
+### Login
+
+Before an application can be created, applicants need to be provided with a username and password to log into the funding portal.  Their user account needs to have the correct permissions, see the section [User Permissions](../administration/user-permissions.md) for more details.
+
+After login in, applicants can change their password at **My account** → **Edit**. All other possible actions can be found on the dashboard page.
+
+## Dashboard
+
+The dashboard is the central page of the funding portal and is usually configured to be the home page. All possible actions that are available to applicants are listed there.
+
+![Dashboard page](.././img/drupal_dashboard.png){ width="500" }
+
+### Application list
+
+The dashboard element **My Applications**  shows the existing applications with some details. Different actions can be performed for applications in this list.
+
+![](../img/drupal_application_list.png)
+
+Applications of funding case type _Sammelantrag_ are listed separately in the dashboard element **My combined Applications**.
+
+#### Create and upload documents
+
+Some actions that are listed in the application list have names of the form **Create: ...**. Those actions lead to pdf documents that were created by the giving organization. They can be downloaded and printed. After filling them out you can upload them to the appropriate place, for example the application or the drawdown form.
```

Comment:
Actually document creation and upload are two different things. They only *might* be used to create documents that are uploaded in the clearing form. However, the created document doesn't need to be determined to be filled out. Document also doesn't directly belong to the *Application list* which is the superior section.

Reply:
Sorry, apparently I oversaw this comment yesterday. I agree with the point that not all documents are determined to be filled out and uploaded. I will change the text accordingly. 
Would it be correct to change the title to "Create documents" and leave it as subsection of "Application list"?  I found the actions "Create: ..." there, that is the reason for the current structure. Or would I omit another place then?

Reply:
I changed the text in this section. Feel free to edit the text further if you want.